pages/common/virsh-help.md

```diff
@@ -0,0 +1,21 @@
+# virsh-help
+
+> Display information about `virsh` commands or command groups.
+> See also: `virsh`.
+> More information: <https://manned.org/virsh>.
+
+- List the `virsh` commands grouped into related categories:
+
+`virsh help`
+
+- List the command categories:
+
+`virsh help | grep "keyword"`
```

Comment:
Not totally sure about this example.  I'd need @sbrl's opinion.

Reply:
Since the output of `virsh help` is rather long and `virsh help | grep "keyword"` brings you to the desired section, I think we can keep itt.